windows-active-directory-mixin/alerts/alerts.libsonnet

```diff
@@ -0,0 +1,62 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'windows-active-directory-alerts',
+        rules: [
+          {
+            alert: 'WindowsActiveDirectoryHighBindOperationsWarning',
+            expr: |||
+              sum without (bind_method, agent_hostname) (increase(windows_ad_binds_total[5m])) / clamp_min(sum without (bind_method, agent_hostname) (increase(windows_ad_binds_total[10m])), 1) * 100 > %(alertsWarningHighBindOperations)s
+            ||| % $._config,
+            'for': '5m',
+            labels: {
+              severity: 'warning',
+            },
+            annotations: {
+              summary: 'There is a high percentage of bind operations. This may indicate brute force attacks or configuration issues.',
```

Comment:
The description for this alert doesn't really match up with reality. The description sounds like we're talking about something like too many bind operations compared to total operations.

Also, does this math make sense? it seems like if there is a single bind operation every minute, (or even every 5 minutes), we're going to end up with 50% which hits the threshhold to trigger the alert?